app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -0,0 +1,61 @@
+// Forked from GOV.UK Frontend, namespace changed to ensure no conflicts.
+
+$app-spacing-scale-0: 0;
+$app-spacing-scale-1: 5px;
+$app-spacing-scale-2: 10px;
+$app-spacing-scale-3: 15px;
+$app-spacing-scale-4: 20px;
+$app-spacing-scale-5: 30px;
+$app-spacing-scale-6: 40px;
+$app-spacing-scale-7: 50px;
+$app-spacing-scale-8: 60px;
+
+// Border widths
+$app-border-width-form-element: 2px;
+
+// Focus
+$app-focus-width: 3px;
```

Comment:
Anything namespaced should have a 'gem-' namespace.

You may be able to reuse https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/helpers/_variables.scss

Reply:
Nice! Done.